docs/pt/docs/fastapi-people.md

```diff
@@ -0,0 +1,178 @@
+# Pessoas do FastAPI
+
+FastAPI possue uma comunidade incrível que recebe pessoas de todos os níveis.
+
+## Criador - Mantenedor
+
+Ei! 👋
+
+Este sou eu:
+
+{% if people %}
+<div class="user-list user-list-center">
+{% for user in people.maintainers %}
+
+<div class="user"><a href="{{ user.url }}" target="_blank"><div class="avatar-wrapper"><img src="{{ user.avatarUrl }}"/></div><div class="title">@{{ user.login }}</div></a> <div class="count">Respostas: {{ user.answers }}</div><div class="count">Pull Requests: {{ user.prs }}</div></div>
+{% endfor %}
+
+</div>
+{% endif %}
+
+Eu sou o criador e mantenedor do **FastAPI**. Você pode ler mais sobre isso em [Help FastAPI - Get Help - Connect with the author](help-fastapi.md#connect-with-the-author){.internal-link target=_blank}.
+
+...Mas aqui eu quero mostrar a você a comunidade.
+
+---
+
+**FastAPI** recebe muito suporte da comunidade. E quero destacar suas contribuições.
+
+Estas são as pessoas que:
+
+* [Help others with issues (questions) in GitHub](help-fastapi.md#help-others-with-issues-in-github){.internal-link target=_blank}.
+* [Create Pull Requests](help-fastapi.md#create-a-pull-request){.internal-link target=_blank}.
+* Revisar Pull Requests, [especially important for translations](contributing.md#translations){.internal-link target=_blank}.
```

Comment:
Missing translation
```suggestion
* [Ajudam outros com issues (dúvidas) no GitHub](help-fastapi.md#help-others-with-issues-in-github){.internal-link target=_blank}.
* [Criam Pull Requests](help-fastapi.md#create-a-pull-request){.internal-link target=_blank}.
* Revisam Pull Requests, [importante especialmente para traduções](contributing.md#translations){.internal-link target=_blank}.
```

Reply:
I didn't translated this links because the page that is referenced is not translated. I think that is better to translate the links when the page is translated to.

Reply:
It makes sense, I didn't think about that